src/tools/sdk/dart_sdk_archive/lib/src/util.dart

```diff
@@ -42,14 +42,19 @@ Map<String, String> archiveMap = {
   'ARMv7': 'arm',
   'ARMv8 (ARM64)': 'arm64',
   'Dart SDK': 'dartsdk',
-  'Dartium': 'dartium'
+  'Dartium': 'dartium',
 };
```

Comment:
Do we need to keep dartium here?

Reply:
Dartium *was* a valid download for a long time.

Although it's expired now.

Maybe it would be nice to drop...

Reply:
Is it not still needed for older builds?